### ISSUE TITLE
symfony example, not working. Permission issue to write session files.

### ISSUE THREAD

**Issue by Tokeiito** [DATA-DOG/dockerfiles#3]
An exception has been thrown during the rendering of a template ("Warning: SessionHandler::read(): open(/var/lib/php/session/sess_i2hkeo73um70mof2q020c5n0t0, O_RDWR) failed: Permission denied (13)") in AcmeDemoBundle::layout.html.twig at line 11.

**Comment by mn7z** [comment]
yep, seems like nginx is running under default user instead of `www`

**Comment by Tokeiito** [comment]
Nginx and PHP-fpm running on www user. From debuging it looks like that something is broken up with /var/lib/php/\* permissions. 
Issue is fixed after you physicaly delete and recreate thous directories inside docker image. But it is not solution.

**Comment by Tokeiito** [comment]
It was solved by moving /var/lib/php/session to volumes and mounting it outside container.
